homeassistant/helpers/intent.py

```diff
@@ -137,7 +137,8 @@ def async_validate_slots(self, slots):
         if self._slot_schema is None:
             self._slot_schema = vol.Schema({
                 key: SLOT_SCHEMA.extend({'value': validator})
-                for key, validator in self.slot_schema.items()})
+                for key, validator in self.slot_schema.items()},
+                                           extra=vol.ALLOW_EXTRA)
```

Comment:
I am a bit surprised this was the fix, as SLOT_SCHEMA already had vol.ALLOW_EXTRAS set.

Reply:
SLOT_SCHEMA is the schema for one specific slot (e.g. name). This sets vol.ALLOW_EXTRAS  to the schema of the parent object containing all slots.